easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.16-foss-2022b-R-4.2.2.eb

```diff
@@ -0,0 +1,1240 @@
+    }),
+    ('AnnotationHub', '3.6.0', {
+        'checksums': ['f6c4062a04a1a8421072ae89cf2ba41ef98c8372cd360f179c89745e0d1660c2'],
+    }),
+    ('AnnotationFilter', '1.22.0', {
+        'checksums': ['d37a0e40e5503c84bc99432c982f4894a10114ae93abe9077074adaeb59b2654'],
+    }),
+    ('ProtGenerics', '1.30.0', {
+        'checksums': ['9116dd55ef2e09f1b1d9d81f4745574d898dcf1d438e58a896357db83e78b7cc'],
+    }),
+    ('ensembldb', '2.22.0', {
+        'checksums': ['51d68978b0f2285be379bbbfbb7b890ac689303434259ef6988a2f5ce3b031ec'],
+    }),
+    ('biovizBase', '1.46.0', {
+        'checksums': ['865393b822d308c96d9987faa0d1e253ad14cf3efd3c3c376896eff2065ea64c'],
+    }),
+    ('OrganismDbi', '1.40.0', {
+        'checksums': ['7cfa7055f5b4625e764ee918e8af46409a7b7f54ad53d3fa4e2268bc38ee8186'],
+    }),
+    ('ggbio', '1.46.0', {
+        'checksums': ['0f983bebac75b2ae957e709d2fa316f547844ceb0136f132e3082ad06df0a3be'],
+    }),
+    ('geneplotter', '1.76.0', {
+        'checksums': ['6a2bf0e14ca16d23c629680967d24c5c08006300483235ee3acceadda6269b24'],
+    }),
+    ('DESeq2', '1.38.3', {
+        'checksums': ['b5db24957eefca903089d9b6ca56db25438a25ad3981ce83fc847eb3647a3e4f'],
+    }),
+    ('ReportingTools', '2.38.0', {
+        'checksums': ['c0673fe479e09590b789ade6d91c50ad7f5e568aa9bc9debf7606301c0b22fdb'],
+    }),
+    ('Glimma', '2.8.0', {
+        'checksums': ['42bcc0390167920fd1b5c58815fcd1beaddaef8aa9aabf315c99f6d1ee5253bb'],
+    }),
+    ('affycoretools', '1.70.0', {
+        'checksums': ['94c4be4223a427763e07bf468e63a1ce2e272a432c5abf4cbd56c2d890a6549f'],
+    }),
+    ('TxDb.Hsapiens.UCSC.hg19.knownGene', '3.2.2', {
+        'checksums': ['063de2b1174782a0b2b8ab7f04a0bdf3c43252cb67c685a9f8ef2b8e318352e9'],
+    }),
+    ('Homo.sapiens', '1.3.1', {
+        'checksums': ['014809fc6ef6410be8dc1094c9cb083719f20d999065ae4bf388855be0913b94'],
+    }),
+    ('BSgenome.Hsapiens.UCSC.hg19', '1.4.3', {
+        'checksums': ['5bfa65d7836449d9b30c356968497cdfaa98be48c4e329e71e8f8a120f3e9d1a'],
+    }),
+    ('AgiMicroRna', '2.48.0', {
+        'checksums': ['c7d5319808ec9267e052fecc1336777d3fa8731efe882554801036d2bcf57060'],
+    }),
+    ('geneLenDataBase', '1.34.0', {
+        'checksums': ['8f9cf0b21de1e9042dd8680c86ca93b72f57c0f68d38acf81a0187cdc2d396f5'],
+    }),
+    ('goseq', '1.50.0', {
+        'checksums': ['c8478273cb97f28a827fca7721e31fa51831d467d292a218540c0f4ad0a60975'],
+    }),
+    ('KEGGgraph', '1.58.3', {
+        'checksums': ['c60fd72bdf5f0fd9335779a15f9f299143cab0592406a418dc643274db3fb2e8'],
+    }),
+    ('GEOquery', '2.66.0', {
+        'checksums': ['7602d0cb9c8d600187d599c2fb6d7328f219193c72c4af1987aaf51968556232'],
+    }),
+    ('rARPACK', '0.11-0', {
+        'checksums': ['c33401e2e31d272d485ce2ed22e7fe43ac641fd7c0a45a9b848d3ad60df1028a'],
+    }),
+    ('mixOmics', '6.22.0', {
+        'checksums': ['3dabec627b5e21da365979d6980565bd338e5a6ec8c3a016f3726ebb6184da70'],
+    }),
+    ('Rhdf5lib', '1.20.0', {
```

Comment:
Failed to build ...

Reply:
maybe it doesn't like HDF5 1.14? Might need to use the compat macros

Reply:
> This R/Bioconductor package provides a mechanism to install the HDF5 system library via an R package.

I am not sure 1.14 is involved here. I fear it is installing an ancient HDF5 version that is the problem. See the repo at https://github.com/grimbough/Rhdf5lib

Reply:
ugh
looks like it ships 1.10.7: https://github.com/grimbough/Rhdf5lib/blob/master/src/Makevars.in

Reply:
Argh. This was a mistake by me. I'd broken my cURL install and this caused the problems here.